app/config.js

```diff
@@ -55,6 +55,9 @@ export function initWalletCfg(testnet, walletPath) {
   if (!config.has("trezor")) {
     config.set("trezor", false);
   }
+  if (!config.has("vsp_is_legacy")) {
+    config.set("vsp_is_legacy", true);
```

Comment:
Thinking if this should be true or false. 

What do you think @alexlyp ?

Reply:
We should be defaulting to use the new VSP ticket purchasing system. (non-legacy)

Reply:
Alright. I've changed it to false and resolved the merge conflicts.